pennylane/gradients/general_shift_rules.py

```diff
@@ -250,11 +229,11 @@ def _combine_shift_rules(rules):
     combined_rules = []
 
     for partial_rules in itertools.product(*rules):
-        c, s = np.stack(partial_rules).T
-        combined = np.concatenate([[np.prod(c)], s])
+        c, *m, s = np.stack(partial_rules).T

```

Comment:
I maybe am not following here, but why `*m`?

Reply:
Because there might not be any multipliers, in which case `m` will be empty.

Reply:
oh! This is a great python hint, all these years and I never realized you could unpack something that isn't there.

Still learning 😎